config.go

```diff
@@ -36,6 +36,10 @@ var (
 	kafkaSslClientKeyPass  = ""
 	kafkaSslCACertFile     = ""
 	kafkaSslValidation     = false
```

Comment:
`kafkaSslValidation` is removed from this suggestion as it is not used anywhere.

https://github.com/Telefonica/prometheus-kafka-adapter/search?q=kafkaSslValidation&type=Code

```suggestion
```